src/color-slider/README.md

```diff
@@ -24,7 +24,7 @@ You can listen to the `input` event and grab the `color` property to get the cur
 ```html
 <color-slider space="hsl"
               stops="red, yellow, lime, aqua, blue, magenta, red"
-              oninput="this.nextElementSibling.textContent = this.color"></color-slider>
+              oncolorchange="this.nextElementSibling.textContent = this.color"></color-slider>
```

Comment:
I'm writing it here since line 22 is not in the PR.

Do I get it correctly that we should mention the `colorchange` event instead of `input` in line 22?

Reply:
Both should be mentioned

Reply:
Oh I see what you mean. Yes, the prose should change